sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts

```diff
@@ -99,24 +99,6 @@ export class KeycloakService {
     return this.getKeyCloakToken(payload);
   }
 
-  /**
-   * Retrieve an token from Keycloak using a provided client id and client secret.
-   * @param clientSecret Client secret.
-   * @param clientId Client ID.
-   * @returns token
-   */
-  public async getTokenFromClientSecret(
```

Comment:
why was this removed

Reply:
Yes it was obsolete and do not have any reference

Reply:
@ann-aot it was not for form.io but it was no longer used.